src/jobs/deploy_env.yml

```
@@ -106,9 +106,26 @@ steps:
   - when:
       condition: <<parameters.slack_notification>>
       steps:
+        - run:
+            name: Slack channels to notify
+            command: |
+              # For prod releases only always notify central dps-releases channel in addition to custom team channels.
+              if [[ "<< parameters.env >>" == "prod" ]] || [[ "<< parameters.env >>" == "production" ]]; then
```

this looks very odd, would be better as 
```suggestion
              if [[ "<< parameters.env >>" == "prod" || "<< parameters.env >>" == "production" ]]; then
```